spec/menu_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'menu'
+
+describe Menu do
+  let(:menu) { Menu.new }
+
+  # instance variables:
+  it { is_expected.to respond_to(:menu_list, :price_list, :dish_list) }
+  # methods:
+  it { is_expected.to respond_to(:print_menu, :load_menu, :prices) }
```

Comment:
When you refactor, you could leave these expectations out if you test their behaviour e.g. line 16 makes it unnecessary to test whether it responds to #menu_list.